### ISSUE TITLE
Images are being rendereds inside anchor tag

### ISSUE THREAD

**Issue by nublson** [9gustin/react-notion-render#110]
I just noticed that the image `<img>` blocks are being rendered inside an anchor `<a>` tag.

I don't know why did you do that but it's hurting my SEO because it's rendering an anchor tag without an href and Lighthouse returns a` Links are not crawlable` error.

Here's how it looks on dev inspector:

```
<a class="rnr-image" target="_blank" rel="noreferrer" >
    <img src="..." alt />
</a>
```

**Comment by 9gustin** [comment]
true! i've added this at one time but i think that it's completely unnecessary, going

**Comment by 9gustin** [comment]
@nublson beta released https://www.npmjs.com/package/@9gustin/react-notion-render/v/3.7.1-beta.0 🚀

**Comment by nublson** [comment]
@9gustin very nice! Thank you!

**Comment by 9gustin** [comment]
Released 🎉 v3.7.1 -> https://www.npmjs.com/package/@9gustin/react-notion-render 😄
